wagtail/admin/static_src/wagtailadmin/js/core.js

```diff
@@ -83,6 +83,57 @@ function enableDirtyFormCheck(formSelector, options) {
 }
 
 $(function() {
+    // Load icon sprite and cache it in local storage.
+    function loadIconSprite() {
+        var spriteURL = wagtailConfig.ADMIN_URLS.SPRITE;
+        var revisionKey = 'spriteRevision';
+        var dataKey = 'spriteData';
+
+        // Feature check
+        if (!document.createElementNS || !document.createElementNS('http://www.w3.org/2000/svg', 'svg').createSVGRect) {
+            return true;
+        }
+
+
+        var isLocalStorage = 'localStorage' in window && window.localStorage !== null;
```

Comment:
I would be more comfortable with this being:

```suggestion
        var isLocalStorage = 'localStorage' in window && typeof window.localStorage !== 'undefined';
```

… so this works in more cases. I’m not too familiar with what `window.localStorage` would register as for browsers that don’t have it, so that check seems more comprehensive.